### DISCUSSION TITLE
Template diracx extension

### DISCUSSION THREAD

**Discussion by ruslan33** [DIRACGrid/diracx#82] [Developer discussions]
Hello,

Following the work done during the 2nd direct hackathon I prepared the template diracx extension project. Please feel free to provide any feedback. Also let me know if you want me to push this to the DIRACGrid area.

https://github.com/ruslan33/mydiracx

Cheers

**Comment by chaen** [comment]
That looks perfect. Can you please create a pull request against diracx repo with that ? Please create the structure as `extensions/src/mydiracx`. 
We will merge it straight away and add tests and the like 
Thanks a lot

**Comment by ruslan33** [comment]
Pull request: https://github.com/DIRACGrid/diracx/pull/124
